wiremock/mappings/ProbationOffenderSearchApi_SearchByNomsNumber.json

```diff
@@ -0,0 +1,156 @@
+{
+  "id": "05c36ca3-f66c-4ffe-917e-65aff9567af2",
+  "request": {
+    "method": "POST",
+    "urlPathPattern": "/search.*",
+    "bodyPatterns": [
+      {
+          "equalToJson": { "nomsNumber": "A1234AI" }
+      }
+  ]
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": [
+      {
+        "offenderId": 1,
+        "softDeleted": false,
+        "otherIds": {
+          "crn": "X320741",
+          "pncNumber": "2018/0123456X",
+          "croNumber": "SF80/655108T",
+          "nomsNumber": "A1234AI",
+          "previousCrn": "X10001"
+        },
+        "title": "Mr",
+        "firstName": "Aadland",
+        "middleNames": [],
+        "surname": "Bertrand",
+        "previousSurnames": [],
+        "dateOfBirth": "1978-01-06",
+        "gender": "Male",
+        "currentDisposal": "1",
+        "currentExclusion": false,
+        "currentRestriction": false,
+        "partitionArea": "some name",
+        "offenderAliases": [
+        ],
+        "contactDetails": {
+          "addresses": [
```

Comment:
Probably it's not significant, but there are a couple of properties which I think are missing from address:

- `id`
- `type`

See example from dev env (crn X349420, noms A5194DY):

```json
"addresses": [
          {
            "id": 2500078722,
            "from": "2020-07-13",
            "noFixedAbode": false,
            "notes": "Comment added by SYSTEM USER on 09/06/2022 at 11:47\nType and Type Verified were set by an automated script based upon the current accommodation Personal Circumstances record on  ( 09-JUN-2022 11:47 )",
            "buildingName": "Tottenham Hotspur Stadium",
            "streetName": "High Street",
            "town": "Lodon",
            "postcode": "N17 1BB",
            "status": {
              "code": "M",
              "description": "Main"
            },
            "type": {
              "code": "A07B",
              "description": "Friends/Family (settled)"
            }
          }
        ]
      },
```